src/main/resources/testng-1.0.dtd

```diff
@@ -62,7 +65,7 @@ Cedric Beust & Alexandru Popescu
     verbose CDATA #IMPLIED
     parallel (false | methods | tests | classes | instances) "false"
     parent-module CDATA #IMPLIED
-    guice-stage CDATA #IMPLIED
+    guice-stage (DEVELOPMENT | PRODUCTION | TOOL) #IMPLIED
```

Comment:
Can we specify a default in the DTD or does it have to be a comment? Either way, yes, specifying the default is the correct way to do this.